src/scene/gsplat/gsplat-compressed-data.js

```diff
@@ -42,7 +42,7 @@ class SplatCompressedIterator {
             }
         };
 
-        const lerp = (a, b, t) => a * (1 - t) + b * t;
+        const lerp = (a, b, t) => a === b ? a : a * (1 - t) + b * t;
```

Comment:
it cold do with a comment, as otherwise it's strange thing to do :)

Reply:
Well only slightly strange. If a === b, then lerping between a and b is always a (or b if you prefer).